uberpoet/loccalc.py

```diff
@@ -0,0 +1,67 @@
+#  Copyright (c) 2021 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import tempfile
+from os.path import join
+
+from .commandlineutil import count_loc
+from .filegen import SwiftFileGenerator
+from .memoize import memoized
+
+
+class LOCCalculator(object):
+
+    def __init__(self):
+        self.swift_gen = SwiftFileGenerator()
+
+    @memoized
+    def calculate_objc_loc(self):
+        raise ValueError("Not supported.")
+
+    @memoized
+    def calculate_swift_loc(self):
+        return self.__calculate_loc(self.swift_gen, 'Swift')
+
+    def __calculate_loc(self, file_gen, language):
```

Comment:
I think one `_` is enough for private funcs in python

Reply:
ha, will change.

Reply:
done